newdle/client/src/components/TopHeader.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {Grid, Header} from 'semantic-ui-react';
+
+import styles from '../App.module.scss';
+
+import Navigator from './Navigator.js';
+
+const steps = [
+  {
+    number: 1,
+    title: 'Choose the Participants',
+    description: 'You can also "Skip" this step and add them later',
+  },
+  {
+    number: 2,
+    title: 'Choose the Timeslots',
+    description: 'Which will be presented as options to the participants',
+  },
+  {
+    number: 3,
+    title: 'Finalize',
+    description: 'Do something final here',
+  },
+];
+
+class TopHeader extends React.Component {
```

Comment:
Let's go for 100% functional components? :wink: I guess it's just a matter of preference but it's a good moment to decide cc/ @indico/development-team

Reply:
yes, no classes please